backend/routes/rides.js

```diff
@@ -146,14 +164,16 @@ router.delete('/:userID/:rideID', async function (req, res, next) {
     } else {//invalid request - rideID not ObjectId
         res.status(400).send("Invalid rideID (not ObjectId) for ride deleting request.");
     }
-})
+});
 
 //cancel a ride
 router.put('/cancel/:userID/:rideID', async function (req, res, next) {
     const driverID = req.params.userID;
     const rideID = req.params.rideID;
     const sameTimeTMR = new Date(new Date().getTime() + 1000 * 60 * 60 * 24);//one day after current time
-    console.log(sameTimeTMR);
+    if (DEBUG) {
+        console.log(sameTimeTMR);
+    }
```

Comment:
```suggestion
```
This line is no longer needed let's get rid of it.

Reply:
Got it